### DISCUSSION TITLE
Welcome to Teal Discussions!

### DISCUSSION THREAD

**Discussion by hishamhm** [teal-language/tl#330] [General]
## 👋 Welcome!

At the community's request (#303), we're enabling Discussions in this repo! We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together :heart:

Of course, if you have any compiler or language issues to report, please continue to use the [issues](https://github.com/teal-language/tl/issues) page for those.

We can use this topic for general introductions, and a discussion on how the community wants to use this space! Welcome!

**Comment by lenscas** [comment]
maybe there should be a place where we can see what kind of posts are meant to go where?

Q&A is easy, just questions about the language
Show and tell is to show of projects written in teal/help with teal (I guess this replaces https://github.com/teal-language/awesome-teal ?)

But I'm not sure what should go in community and general?

General being discussions about the teal project, I guess?

**Comment by hishamhm** [comment]
> Show and tell is to show of projects written in teal/help with teal (I guess this replaces https://github.com/teal-language/awesome-teal ?)

I think it can feed awesome-teal with material, but it's nice to have the short summary in the awesome repo, and also have the possibility of forum-style discussion about new projects in Show and Tell.

> But I'm not sure what should go in community and general?

I did write some one-line descriptions about these topics (which I slightly adapted from the default category list), but they're a bit hidden in the UI — the only place I found where they show up is when you click "New discussion" and then open the "Category..." dropdown:

![image](https://user-images.githubusercontent.com/245621/104264771-02caa180-546b-11eb-94d5-2a355028e1db.png)

so, in short, this is what I had in mind initially:

* Community: links to resources, events, meetups, etc.
* General: general talk about Teal

How does that sound?

**Reply by lenscas** [reply]
ah, I did not see those descriptions, as I was just looking at the sidebar thingy. 

Looks good to me

**Reply by tsafin** [reply]
An in any case, it's not a big deal if anyone would misattribute his post. Admin could change category, convert it to issue or back to discussions post.
